src/client/css/partials/error.css

```diff
@@ -0,0 +1,24 @@
+.error-page {
```

Comment:
Do we also need the license header for stylesheets?

Reply:
I just spent too much time trying to set up a linting rule for this, but something in our linting/VSCode setup keeps making it remove arbitrary lines and it's getting annoying, so I gave up. In any case, most CSS files don't have it, and AFAIK the licence doesn't require it, so I'm giving up at this point.